src/server_manager/electron_app/package_linux.action.sh

```diff
@@ -16,7 +16,7 @@
 
 source src/server_manager/scripts/fill_packaging_opts.sh "$0" "$@"
 
-yarn 'do' server_manager/electron_app/build
+npm run 'action' server_manager/electron_app/build
```

Comment:
These single-quotes were just to keep shellcheck happy because `do` is a keyword in shell.  You can remove them.